multixstar_logging.py

```diff
@@ -195,11 +195,11 @@ def main(argv=None):
     check_enviroment(workDir)
 
     wdir = "mxstar." + str(get_sufix(workDir))
-    os.mkdir(wdir)
-    os.chdir(wdir)
     if not workDir[-1] == "/":
```

Comment:
We can probably remove most of the ` path_var[-1] == "/"`  parts of the code if we change the ` + ` between strings to `path.join` not something that you have to do just an idea.